packages/design-system-dashboard-cli/src/find-ds-components.js

```diff
@@ -210,7 +206,9 @@ function findComponents(searchStrings) {
   );
 
   const wcTagRegex = /<(va-[^\s>]+)/gms;
-  const wcUswds3Regex = /<(Va[^\s]+|va-[^\s]+)(\s|\n)[^>]*?uswds/gms;
+  // Excludes uswds="false" (explicit opt-out) == only matches true if V3, else not V3
+  const wcUswds3Regex =
+    /<(Va[^\s]+|va-[^\s]+)(\s|\n)[^>]*?uswds(?!=?"?'?false)/gms;
```

Comment:
nice addition here! When I added this a couple months ago, I definitely didn't consider the explicit opt out. Even more important right now since we're going to be making v3 the default here soon.